x-pack/plugin/migrate/src/test/java/org/elasticsearch/xpack/migrate/action/ReindexDatastreamIndexRequestTests.java

```diff
@@ -19,11 +22,19 @@ protected Writeable.Reader<Request> instanceReader() {
 
     @Override
     protected Request createTestInstance() {
-        return new Request(randomAlphaOfLength(20));
+        return new Request(randomAlphaOfLength(20), randomAPIBlockSet());
     }
 
     @Override
     protected Request mutateInstance(Request instance) {
-        return new ReindexDataStreamIndexAction.Request(randomValueOtherThan(instance.getSourceIndex(), () -> randomAlphaOfLength(20)));
+        return new ReindexDataStreamIndexAction.Request(
```

Comment:
I think you just want to change one thing at a time here, so that we know that any single change causes the objects to not be equal.